_includes/head.html

```diff
@@ -10,7 +10,7 @@
 <meta name="robots" content="nocache">
 {% endif %}
 <title>{{ page.title }} | {{ site.site_title }}</title>
-<link rel="canonical" href="{{ page.canonical | absolute_url }}">
```

Comment:
Don't we want the canonical url to alway point to the "stable" version of the docs? With this change, the canonical is always matching the current docs version...

Reply:
I thought Dan said we would want to only reference the numbered versions. I might have misunderstood! I can change back.

Reply:
Yes, I think that's right, but only the canonical number versions, so the equivalent of /stable. Right now, that would make it would always be v20.2. That's my understanding at least. Good to check with Dan.

Reply:
I see, you're saying we want all versions of the same page to now have canonical links to `v20.2`, which we then update to `v21.1` and so on. If we have time on tomorrow's call to clarify with Dan that would be great.

Reply:
Done